includes/Dashboard/Templates/Withdraw.php

```diff
@@ -533,4 +560,178 @@ public function user_cancelled_withdraws( $user_id ) {
+            $message = __( 'You don\'t have sufficient balance for a withdraw request!', 'dokan-lite' );
+            $this->show_error_messages( $message );
+            return;
+        }
+
+        $payment_methods         = array_intersect( dokan_get_seller_active_withdraw_methods(), dokan_withdraw_get_active_methods() );
+        $default_withdraw_method = dokan_withdraw_get_default_method( $current_user_id );
+        dokan_get_template_part( 'withdraw/request-form', '', array(
+            'amount'          => NumberUtil::round( $balance, wc_get_price_decimals(), PHP_ROUND_HALF_DOWN ), // we are setting 12.3456 to 12.34 not 12.35
+            'withdraw_method' => $default_withdraw_method,
+            'payment_methods' => $payment_methods,
+        ) );
+    }
+
+    /**
+     * Get pending withdraw request in dashboard listing.
+     *
+     * @since 3.3.1
+     *
+     * @return void
+     */
+    public function pending_withdraw_requests() {
+        if ( ! current_user_can( 'dokan_manage_withdraw' ) ) {
+            return;
+        }
+
+        $user_id           = dokan_get_current_user_id();
+        $withdraw_requests = dokan()->withdraw->get_withdraw_requests( $user_id );
+
+        dokan_get_template_part( 'withdraw/pending-request-listing-dashboard', '', array(
+            'withdraw_requests' => $withdraw_requests,
+        ) );
+    }
+
+    /**
+     * Add withdraw listing query var
+     *
+     * @since 3.3.1
+     *
+     * @param array $query_vars
+     *
+     * @return array
+     */
+    public function add_withdraw_list_query_var( $query_vars ) {
+        $query_vars[] = 'withdraw-requests';
+
+        return $query_vars;
+    }
+
+    /**
+     * Display withdraw listing.
+     *
+     * @since 3.3.1
+     *
+     * @param array $query_vars
+     *
+     * @return void
+     */
+    public function display_request_listing( $query_vars ) {
+        if ( empty( $query_vars ) || ! array_key_exists( 'withdraw-requests', $query_vars ) ) {
+            return;
+        }
+
+        if ( ! current_user_can( 'dokan_view_withdraw_menu' ) ) {
+            dokan_get_template_part( 'global/no-permission' );
+        } else {
+            dokan_get_template_part( 'withdraw/withdraw' );
+        }
+    }
+
+    /**
+     * Set withdraw menu as active.
+     *
+     * @since 3.3.1
+     *
+     * @param string $active_menu
+     * @param $request
+     * @param array $active
+     *
+     * @return string
+     */
+    public function active_dashboard_nav_menu( $active_menu, $request, $active ) {
+        if (  'withdraw-requests' !== $active_menu ) {
+            return $active_menu;
+        }
+        return 'withdraw';
+    }
+
+    /**
+     * Redirect to vendor dashboard.
+     *
+     * @since 3.3.1
+     *
+     * @return void
+     */
+    public function redirect_to_dashboard() {
+        if ( preg_match( '/\/withdraw(-requests)?\//i', sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) ) ) {
```

Comment:
this will conflict with wpml, kindly rethink the logic

Reply:
Changed as per your recommendation.